pkg/serverless/trace/inferredspan/constants.go

```diff
@@ -11,6 +11,7 @@ const (
 	apiName          = "apiname"
 	bucketARN        = "bucket_arn"
 	bucketName       = "bucketname"
+	busName          = "BusName"
```

Comment:
this should comply with [Consistent APM span tags for AWS requests](https://docs.google.com/document/d/1450pjyhzqHTTtEvPyZGzH1cKpca7gOwtxgel_LUsR4w/edit#heading=h.w9tpabmagxvr)

Reply:
Yeah I've renamed the custom fields to `x-datadog-sent-timestamp` and `x-datadog-bus-name`, and put them under `_datadog`. What do you think of this approach?

Reply:
Where do those fields come from?

Reply:
I just came up with them on my own. There's no existing naming convention for these fields. I tried to follow the style of the [existing Datadog headers](https://docs.datadoghq.com/tracing/trace_collection/trace_context_propagation/?tab=java#datadog-format)